Extensions/people_notifier.js

```diff
@@ -1,8 +1,8 @@
 //* TITLE Blog Tracker **//
-//* VERSION 0.4.1 **//
+//* VERSION 0.5.0 **//
 //* DESCRIPTION Track people like tags **//
 //* DEVELOPER STUDIOXENIX **//
-//* DETAILS Blog Tracker lets you track blogs like you can track tags. Add them on your dashboard, and it will let you know how many new posts they've made the last time you've checked their blogs, or if they've changed their URLs. **//
+//* DETAILS Blog Tracker lets you track blogs like you can track tags. Add them on your dashboard, and it will let you know how many new posts they've made the last time you've checked their blogs, or if they've changed their URLs.<br><br>Please be aware that the more blogs you add, the longer it will take to track them all. **//
```

Comment:
My impression is that extension details aren't commonly read, so bolding this warning might not catch too much attention. However, what I could do is also include this warning in the window that pops up when adding a new blog. I think users are more likely to see that than read the extension details.